ykcapi/src/lib.rs

```diff
@@ -102,11 +102,12 @@ impl Registers {
                 id
             )
         }
-        let mut val = self.read_from_stack(id.try_into().unwrap());
+        let val = self.read_from_stack(id.try_into().unwrap());
         // Due to the return address being pushed to the stack before we store RSP, its value is
         // off by 8 bytes.
         if id == 7 {
-            val += std::mem::size_of::<usize>();
+            todo!(); // Check this is still true now that llvm_deoptimize is a naked function.
+            // val += std::mem::size_of::<usize>();
```

Comment:
Commented code.

Reply:
That was on purpose as I'm not sure yet if this is still needed. Although you're probably gonna tell me to remove it nonetheless.

Reply:
`todo`s I like -- keep that! Commented code I don't like -- remove that!

Reply:
Fixed in 08d7ccb